src/App/App.xaml.cs

```diff
@@ -140,6 +140,15 @@ public App(AppOptions appOptions)
                         }
                     });
                 }
+                else if (message.Command == "removeMasterPassword")
```

Comment:
Message command should be consistent with jslib (`convertAccountToKeyConnector`)

Reply:
still valid?

Reply:
68267be